change/@microsoft-teams-js-4b8c81b9-6691-4b9d-bb1a-f8cb66a0c130.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Adding isSupported check and missing unit test cases",
```

Comment:
Can you please add for which capability this change is adding the unit test cases/isSupported check?